wallet/src/actors/worker/error.rs

```diff
@@ -44,6 +44,8 @@ pub enum Error {
         _0, _1
     )]
     NodeBehindLocalTip(u32, u32),
+    #[fail(display = "provided birth_date epoch is greater than the current: {}", _0)]
```

Comment:
I'd even go further and be explicit about both values: 
```suggestion
    #[fail(display = "The provided `birth_date` epoch is greater than the current epoch ({} > {})", _0, _1)]
```